### ISSUE TITLE
Language Server never finishes starting, REPL extremely slow to start.

### ISSUE THREAD

**Issue by SantiRaposo** [julia-vscode/julia-vscode#910]
I'm using 0.13rc5, and Julia 1.3. Once i open a .jl file, the only thing that appears in the output of the Julia Language Server is

```
[ Info: Started symbol server
[ Info: CSV not stored on disc
```

And If i try to execute code using `Alt + Enter` it just displays the message that I have to wait until the Language Server starts.

Executing code with `Ctrl + Enter` works, although the REPL feels very slow to start up, after sending one line of code it takes almost a minute until it is executed and the REPL becomes responsive again.

The output of `versioninfo()` is the following:

```
julia> versioninfo()
Julia Version 1.3.0
Commit 46ce4d7933 (2019-11-26 06:09 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin19.0.0)
  CPU: Intel(R) Core(TM) i5-4278U CPU @ 2.60GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, haswell)
Environment:
  JULIA_EDITOR = "/Applications/Visual Studio Code.app/Contents/Resources/app/bin/code"
  JULIA_NUM_THREADS = 1
```

My VScode version is 1.41.0

**Comment by davidanthoff** [comment]
The language server is indexing all your packages, and that is taking a long time. It should be a onetime thing, i.e. once it is done, we cache the results on disc and then startup should be much better. But be warned, that initial indexing can take a long time.

My guess is that the REPL being slugish is related to that. If you still see that once the indexing is finished, please open a new issue! I'm going to close this here because we are aware of the long startup problem and tracking that elsewhere already. Thanks for the report!
